airtable/plugin.py

```diff
@@ -88,11 +119,12 @@ class AirtableRecord:
     fields: dict[str, Any]
 
 
-def airtable_record_list(base_id: str, table_id_or_name: str) -> list[AirtableRecord]:
+def airtable_record_list(airtable_path: AirtablePath) -> list[AirtableRecord]:
```

Comment:
Restore this and the functions below to use IDs/names, as I think that is more clear (since "path" is not really a concept anywhere).